client/src/components/Swap/OnRamp.tsx

```diff
@@ -70,8 +71,8 @@ export const OnRamp: React.FC<OnRampProps> = ({
   const {
     data: verifyProofRaw,
   } = useContractRead({
-    address: sendProcessorAddress,
-    abi: sendProcessorAbi,
+    address: sendProcessorAddress ?? undefined,
+    abi: sendProcessorAbi ?? undefined,
```

Comment:
Is there a potential for a runtime error? If not I'd rather leave these until we address them all